app/assets/stylesheets/partials/custom.scss

```diff
@@ -32,7 +33,7 @@ a:active {
 }
 
 .easy-block-v1 .overflow-h h3 {
-  width: 60%; 
+  width: 60%;
```

Comment:
Properties should be ordered float, font-size, line-height, margin-bottom, width